### DISCUSSION TITLE
False Positives

### DISCUSSION THREAD

**Discussion by henroFall** [jef/streetmerchant#1854] [Help]
I've been looking to find a discussion on false positives but can't see one. ... Is there one? I've noticed the same 3070 popping up at best buy and microcenter but it's not real. I'm filtering them out with my price limit but I'm afraid I'll miss real cards this way.

Is there a way to block a particular hit, or is there a means to report false positives, or is this just the way it is?

These are the hits:
[bestbuy] [pny (3070)] xlr8 revel
[microcenter] [asus (3070)] strix oc 

Thanks!

**Comment by joshuac-dev** [comment]
Having the same problem with Newegg.  Did you ever find an answer? @henroFall

**Reply by henroFall** [reply]
No, I didn't. There are a couple of repeated false positives from bestbuy and microcenter that I have to filter out by setting the min price so high that they just pop up on screen but do not push to my notifier (pushbullet). 

The app did work for me in that it popped the other day with a bunch of cards that showed up at Microcenter here, but there's a homeless shelter full of people camping there in hopes of getting one of a few vouchers that they hand out in the morning before opening. So I missed those.
